website/docs/docs/build/snapshots.md

```diff
@@ -18,215 +18,195 @@ Snapshots implement [type-2 Slowly Changing Dimensions](https://en.wikipedia.org
+- A number of other configurations are also supported (for example, `tags` and `post-hook`). For the complete list, refer to [Snapshot configurations](/reference/snapshot-configs).
+- You can configure snapshots from both the `dbt_project.yml` file and a `config` block. For more information, refer to the [configuration docs](/reference/snapshot-configs).
 
-1. Create a file in your `snapshots` directory with a `.sql` file extension, e.g. `snapshots/orders.sql`
-2. Use a `snapshot` block to define the start and end of a snapshot:
 
-<File name='snapshots/orders_snapshot.sql'>
+### Add a snapshot to your project
 
-```sql
-{% snapshot orders_snapshot %}
+To add a snapshot to your project follow these steps. For users on versions 1.8 and earlier, refer to [Configure snapshots in versions 1.8 and earlier](#configure-snapshots-in-versions-18-and-earlier). 
 
-{% endsnapshot %}
-```
+1. Create a YAML file in your `snapshots` directory: `snapshots/orders_snapshot.yml` and add your configuration details. You can also configure your snapshot from your `dbt_project.yml` file ([docs](/reference/snapshot-configs)).
 
-</File>
+    <File name='snapshots/orders_snapshot.yml'>
 
-3. Write a `select` statement within the snapshot block (tips for writing a good snapshot query are below). This select statement defines the results that you want to snapshot over time. You can use `sources` and `refs` here.
+    ```yaml
+    snapshots:
+      - name: orders_snapshot
+        relation: source('jaffle_shop', 'orders')
+        config:
+          schema: snapshots
+          database: analytics
+          unique_key: id
+          strategy: timestamp
+          updated_at: updated_at
 
-<File name='snapshots/orders_snapshot.sql'>
+    ```
+    </File>
 
-```sql
-{% snapshot orders_snapshot %}
+2. Since snapshots focus on configuration, the transformation logic is minimal. Typically, you'd select all data from the source. If you need to apply transformations (like filters, deduplication), it's best practice to define an ephemeral model and reference it in your snapshot configuration.
 
-select * from {{ source('jaffle_shop', 'orders') }}
+    ```yaml
+    -- models/ephemeral_orders.sql
+    {{ config(materialized='ephemeral') }}
```

Comment:
For consistency with the other examples, I'd suggest using `<File ...>...</File>` for the file named `models/ephemeral_orders.sql` and then remove the comment containing the file name.
`